app/app.rb

```diff
@@ -0,0 +1,79 @@
+ENV["RACK_ENV"] ||= 'development'
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative 'data_mapper_setup'
+
+class Chitter < Sinatra::Base
+  use Rack::MethodOverride
+  enable :sessions
+  set :session_secret, 'shhh'
+  register Sinatra::Flash
+
+  get '/peeps' do
+    @peeps = Peep.all
+    erb :'peeps/index'
+  end
+
+  get '/peeps/new' do
+    erb :'peeps/new'
+  end
+
+  post '/peeps' do
+    peep = Peep.create(message: params[:message])
+    params[:tags].split(', ').each do |tag|
+      peep.tags << Tag.first_or_create(name: tag)
+    end
+    peep.save
+    redirect to '/peeps'
+  end
+
+  get '/peeps/:name' do
```

Comment:
I misread this code and thought it was about showing the peeps for a certain user.  Could the `:name` variable be named more clearly?